XCUITest/SettingAppearanceTest.swift

```diff
@@ -98,6 +98,7 @@ class SettingAppearanceTest: BaseTestCase {
         let reviewCell = app.cells["settingsViewController.rateFocus"]
         let safariApp = XCUIApplication(privateWithPath: nil, bundleID: "com.apple.mobilesafari")!
         
+        app.tables.firstMatch.swipeUp()
```

Comment:
Necessary to work on iPhone 8 (the only reason this test was failing was because the cell was hidden below the fold).